### PR TITLE
remove $ from code-blocks in build_all.sh

### PR DESCRIPTION
**Pull request by sharunkumar**
Follow up for https://github.com/google/comprehensive-rust/issues/334#issuecomment-1630276602

### CODE REVIEW

File: src/android/aidl/client.md [hunk 1]
```diff
@@ -20,5 +20,8 @@ Build, push, and run the client on your device:
 
 ```shell
 {{#include ../build_all.sh:birthday_client}}
+```
+
+```shell
```
Comment (mgeisler) [inline src/android/aidl/client.md:25]: Nice! Could you remove the `shell` language here and elsewhere — the output is just text and a stray `$` should not be syntax highlighted.

Comment (mgeisler) [reply src/android/aidl/client.md:25]: Sorry, I only saw [your comment now](https://github.com/google/comprehensive-rust/issues/334#issuecomment-1631580582) — please change `shell` to `text`. The default language is `rust` and so `mdbook test` will try to treat the code block as Rust code if you don't explicitly make it `text`.

Comment (sharunkumar) [reply src/android/aidl/client.md]: Done ✌️

**Review by mgeisler — APPROVED** [review]
Thanks for the help!

**Pull request merged**